src/ResourceManagement/Compute/Domain/IVirtualMachineScaleSetsBeta.cs

```diff
@@ -37,7 +37,7 @@ public interface IVirtualMachineScaleSetsBeta :
         /// <param name="vmId">The virtual machine instance id.</param>
         /// <param name="inputCommand">Command input.</param>
         /// <return>Handle to the asynchronous execution.</return>
-        Task<Models.RunCommandResultInner> RunCommandVMInstanceAsync(string groupName, string scaleSetName, string vmId, RunCommandInput inputCommand, CancellationToken cancellationToken = default(CancellationToken));
+        Task<IRunCommandResult> RunCommandVMInstanceAsync(string groupName, string scaleSetName, string vmId, RunCommandInputInner inputCommand, CancellationToken cancellationToken = default(CancellationToken));
```

Comment:
same comment as before for exposing `RunCommandInput` then translating that to `RunCommandInputInner` internally.